Birthday-2024-Project/Scripts/GridLogic.gd

```diff
@@ -29,6 +51,72 @@ func SetupLevel(levelSetupData : LevelSetup):
 		# 	#set piece to outskirts of level grid
 	pass
 
-# # Called every frame. 'delta' is the elapsed time since the previous frame.
-# func _process(delta):
-# 	pass
+#returns an integer; 0 is empty, 1 is occupied, 2 is blocked or unused
+func GetGridSpace(x : int, y : int):
```

Comment:
since you even left a comment above to give the return type:
`func GetGridSpace(x: int, y: int) -> int:`